Assets/Scripts/Requests/Requests.cs

```diff
@@ -42,6 +42,7 @@ public void Add(Request request)
         public void Remove(Guild guild)
         {
             _requests[guild].Complete();
+            Manager.Stats.RequestsCompleted++;
```

Comment:
Can a quest get removed without being completed? I don't believe so, just want to consider potential edge cases.

Reply:
Nope, the only time this remove method is called is the request completed event outcome